resources/cluster-essentials/templates/microfrontend.crd.yaml

```diff
@@ -1,34 +1,34 @@
 apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
-  name: "microfrontends.ui.kyma-project.io"
+  name: 'microfrontends.ui.kyma-project.io'
 spec:
   group: ui.kyma-project.io
   version: v1alpha1
   scope: Namespaced
   names:
-    plural: "microfrontends"
-    singular: "microfrontend"
+    plural: 'microfrontends'
+    singular: 'microfrontend'
```

Comment:
Applications is a cluster MicroFrontend, not a regular MicroFrontend, so wrong file. Additionally, we probably should add docs for that if we change the resource definition. 
But before that, please as Philipp if that should be an 'official' feature, since for now it's only relevant for Applications. If not, then maybe we don't need to edit the CRDs and add docs at all.